pkg/cloud/services/network/natgateways.go

```diff
@@ -106,12 +106,18 @@ func (s *Service) reconcileNatGateways() error {
 			}
 		}
 		ngws, err := s.createNatGateways(subnetIDs)
+		natGatewaysIPs := []string{}
```

Comment:
```suggestion
		var natGatewaysIPs []string
```

Reply:
Otherwise, you'll get an empty list but a list nevertheless. It might make things awkward in certain places.

Reply:
Done